oneclick/routes.py

```diff
@@ -0,0 +1,107 @@
+from transbank.error.transbank_error import TransbankError
+from transbank.oneclick.request import MallTransactionAuthorizeDetails
+
+from oneclick import bp
+from flask import render_template, request
+from transbank.oneclick.mall_inscription import MallInscription
+from transbank.oneclick.mall_transaction import MallTransaction
+import random
+
+
+@bp.route('start-form', methods=['GET'])
```

Comment:
you should remove the `-form` since a 'GET' request will work

Reply:
fixed